swagger.yaml

```diff
@@ -5428,6 +5430,34 @@ definitions:
     items:
       type: string
       pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]|[1-2][0-9]|3[0-2]?$'
+  
+  download_boot_artifacts_request:
+    type: object
+    description: Information sent to the agent for downloading artifacts to boot a host into discovery.
+    required:
+      - kernel_url
+      - rootfs_url
+      - initrd_url
+    properties:
+      kernel_url:
+        type: string
+        description: URL address to download the kernel.
+      rootfs_url:
+        type: string
+        description: URL address to download the rootfs.
+      initrd_url:
+        type: string
+        description: URL address to download the initrd.
+
+  download_boot_artifacts_response:
+    type: object
+    description: The result of downloading artifacts to a host.
+    required:
+      - error
+    properties:
+      error:
```

Comment:
Do we think it's worth creating this struct if it just has an error string?
The reply already has an `error` field.

I guess this is better if we ever wanted to expand it? @avishayt what do you think?

Reply:
As I wrote above, I think it's not necessary.  Better to not redefine "error" here, and no need to define an empty struct.  We can always create it in the future if it becomes necessary and use agent-upgrade.

Reply:
Makes sense we can use the pre-existing step-reply arguments. Thank you for your input!

@carbonin if you agree, I'll go ahead and remove this model